eslint.config.mjs

```diff
@@ -346,31 +346,31 @@ export default [
       'error',
       {
         name: '__filename',
-        message: 'Use import.meta.url instead',
+        message: 'Use import.meta.url instead.',
       },
       {
         name: '__dirname',
-        message: 'Not available in ESM',
+        message: 'Not available in ESM.',
       },
       {
         name: 'exports',
-        message: 'Not available in ESM',
+        message: 'Not available in ESM.',
       },
       {
         name: 'module',
-        message: 'Not available in ESM',
+        message: "'module' must be imported in ESM.",
```

Comment:
```suggestion
        message: "Not available in ESM.",
```

Reply:
Isn't module importable?
```mjs
import module from 'module';
console.log(module)
```

Reply:
This has nothing to do with the `module` "global-like" identifier in the context of a CJS module.